services/Meetup.json

```diff
@@ -3,17 +3,11 @@
     "documents": {
         "tos": {
             "location": "https://help.meetup.com/hc/en-us/articles/360027447252",
-            "contentSelector": ".article",
-            "filters": [
-                "removeFooter"
-            ]
+            "contentSelector": ".article-info"
         },
         "privacy": {
             "location": "https://help.meetup.com/hc/en-us/articles/360044422391",
-            "contentSelector": ".article",
-            "filters": [
-                "removeFooter"
-            ]
+            "contentSelector": ".article-info"
```

Comment:
This selector removes the document title. The previous one was better.

Reply:
Ok thanks, thought the title wasn't necesary, I reverted the commit !